### PR TITLE
:building_construction: ADD swapflow

### PR DESCRIPTION
**Pull request by stefanradouane**
- CHANGE image to other folder

### CODE REVIEW

File: plantswap/components/LangChanger/LangChanger.js [hunk 1]
```diff
@@ -2,17 +2,25 @@
...
+        next={() =>
+          router.replace(
+            locale === "nl" ? `/en${currentPage}` : `/nl${currentPage}`
+          )
+        }>
         {dictionary.language.switch}
-      </Link>
+      </Button>
+      {/* <Link href={locale === "nl" ? `/en${currentPage}` : `/nl${currentPage}`}> */}
+      {/* </Link> */}
```
Comment (stefanradouane) [inline plantswap/components/LangChanger/LangChanger.js:23]: Button component is almost the same as the Link component.
Component isn't styled yet.

File: plantswap/components/Uploader/Uploader.js [hunk 2]
```diff
@@ -9,118 +9,144 @@ import Title from "../Title/Title";
...
-        e.preventDefault();
-        setImage(undefined);
-        setUploadedValue(0);
-        setData(undefined);
-        setShowData(false);
-      }
     }
   };
 
+  console.log(currentFetch);
```
Comment (Marengd) [inline plantswap/components/Uploader/Uploader.js:83]: Spotted a console.log here. It's usually best to remove these before production for cleaner, more efficient code.

File: plantswap/src/app/[lang]/planten/page.js [hunk 3]
```diff
@@ -3,13 +3,24 @@ import Title from "../../../../components/Title/Title.js";
...
+// console.log(dataset);
 
 export default async function Home({ params }) {
   const dictionary = await getDictionary(params.lang);
   return (
-    <Layout locale={params.lang} dictionary={dictionary}>
+    <Layout locale={params.lang} dictionary={dictionary} noFooter={true}>
       <main>
-        <Title title={"h1"}>Hier komt een overzicht van alle planten</Title>
+        {/* <Title title={"h1"}>Hier komt een overzicht van alle planten</Title> */}
```
Comment (Marengd) [inline plantswap/src/app/[lang]/planten/page.js:17]: Found some commented- out code here. To keep our codebase neat and maintainable, it's often best to remove these.

File: plantswap/components/Switcher/switcher.module.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
+.plantswitcher {
+  --count: 2;
+  // padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  // background-color: #bababa; // fixme
+
+  &__footer {
+    color: black;
```
Comment (Mennovlaming) [inline plantswap/components/Switcher/switcher.module.scss:10]: var(--color-black)

**Review by Marengd — COMMENTED** [review]
Feedback referring to console.logs and commented code.

File: plantswap/components/SwapFlow/SwapFlow.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  useEffect(() => {
+    sessionStorage.setItem("flowData", JSON.stringify(flowData));
+  }, [flowData]);
+
+  return (
+    <section style={{ padding: "1rem" }}>
+      <SwapFlowReturn
+        flowData={flowData}
+        setFlowData={setFlowData}
+        totalSteps={totalSteps}
```
Comment (davidsd29) [inline plantswap/components/SwapFlow/SwapFlow.js:46]: Check what the fucntion is of 'totalSteps. You're currently not using it

File: plantswap/components/Switcher/Switcher.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import Text from "../Text/Text";
+import Title from "../Title/Title";
+import styles from "./switcher.module.scss";
+// import dataset from "../../src/app/[lang]/swap/data.json"
```
Comment (davidsd29) [inline plantswap/components/Switcher/Switcher.js]: unnecessary commented code

**Review by davidsd29 — APPROVED** [review]

**Pull request merged**